src/simulator_core/solver/network/assets/base_node_item.py

```diff
@@ -57,3 +57,11 @@ def connect_asset(self, asset: BaseItem, connection_point: int) -> None:
         :param BaseItem asset: The asset to connect to the node item.
         :param int connection_point: The connection point of the asset to connect to.
         """
+
+    @abstractmethod
+    def get_connected_assets(self) -> list[tuple[BaseItem, int]]:
```

Comment:
Is er een reden om deze method "get_connected_asses" te noemen i.p.v. "get_connected_nodes"?

Reply:
Method naming is correct, will return the connected assets as a tuple